specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,670 @@
+            "description": "An error response received from the Policy store service",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataPolicy_Unpublish": {
+            "$ref": "./examples/DataPolicy_UnPublish.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DataPolicyList": {
+      "required": [
+        "value"
+      ],
+      "description": "A paginated list of purview data policies.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "A list of purview data policies",
+          "items": {
+            "$ref": "#/definitions/DataPolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Pagination link"
+        }
+      }
+    },
+    "DataPolicy": {
+      "required": [
+        "name",
+        "decisionRules",
+        "members",
+        "description"
+      ],
+      "type": "object",
+      "description": "Purview data policy",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The policy name"
+        },
+        "state": {
+          "description": "The state of the policy",
+          "enum": [
+            "Published",
+            "Draft"
+          ],
+          "readOnly": true,
+          "type": "string",
+          "x-ms-enum": {
+            "name": "state",
+            "modelAsString": true
+          }
+        },
+        "description": {
+          "type": "string",
+          "description": "The policy description"
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The etag version of a policy"
+        },
+        "scopes": {
+          "type": "array",
+          "readOnly": true,
+          "description": "Array of data source scopes (ARM resourceIds) where the policy is published",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        },
+        "members": {
+          "type": "object",
+          "description": "Members of the policy",
+          "$ref": "#/definitions/PolicyMembers"
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        },
+        "decisionRules": {
+          "type": "array",
+          "description": "Array of decisionRules for the policy",
+          "items": {
+            "$ref": "#/definitions/PolicyDecisionRule"
+          }
+        }
+      }
+    },
+    "PolicyMembers": {
+      "type": "object",
+      "description": "Policy member",
+      "properties": {
+        "entraMembers": {
```

Comment:
What is "entra"?

Reply:
Azure active directory is renamed to `entra`. So, PMs are asking us to rename `aadMembers` field as `entraMembers`.
@inward-eye can you please elaborate?

Reply:
I see. Thanks.
https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/new-name